pkg/api-server/dataplane_overview_endpoints.go

```diff
@@ -112,8 +109,9 @@ func (r *dataplaneOverviewEndpoints) inspectDataplanes(request *restful.Request,
 		rest_errors.HandleError(response, err, "Could not retrieve dataplane overviews")
 		return
 	}
+	namePrefix := request.QueryParameter("name")
```

Comment:
I think this query param should be `name-prefix` or `name_prefix`.
1) I'd expect that name query param would query exact name
2) if not, I'd expect this would be a partial match to any part of a string, not prefix.

Reply:
or namePrefix?